features/facebookImport/src/popUps/infoPopUps/messages.jsx

```diff
@@ -1,35 +1,40 @@
 import React from "react";
 
 import i18n from "../../i18n.js";
-import BaseInfoPopUp from "../baseInfoPopUp/baseInfoPopUp.jsx";
 import Infographic from "../../components/infographic/infographic.jsx";
 
 const MessagesInfoPopUp = () => {
-    const messagesInfoText = [
+    return (
         <>
             <p>{i18n.t("messagesInfoScreen:text1")}</p>
-            <Infographic
-                type="messagesBarChart"
-                texts={{
-                    text1: i18n.t("infographics:messagesBarChart.text1"),
-                    text2: i18n.t("infographics:messagesBarChart.text2"),
-                    bigbold1: i18n.t("infographics:messagesBarChart.bigbold1"),
-                    bold2_1: i18n.t("infographics:messagesBarChart.bold2"),
-                    bold2_2: i18n.t("infographics:messagesBarChart.bold2"),
-                    bold2_3: i18n.t("infographics:messagesBarChart.bold2"),
-                    bold2_4: i18n.t("infographics:messagesBarChart.bold2"),
-                }}
-            />
+            <div className="full-width">
```

Comment:
You seem to be using this div repeatedly with `InfoGraphic`. Can't you just adapt the style to make it full-width? Or as an option, if that's not always the case.

Reply:
That would have been the smart way of going about it 🤦 Will try to fix with the next commit.